templates/base.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+  <title>Taxi Service</title>
+</head>
+<body>
+    <div class="sidebar">
+        {% block sidebar %}
+            {% include "includes/sidebar.html" %}
+        {% endblock %}

```

Comment:
it seems to me that sidebar shouldn't be placed at the same level as content, move it to some right/left side of a page

Reply:
done